sdk/java-v2/src/main/java/org/arvados/client/logic/keep/FileDownloader.java

```diff
@@ -70,6 +73,34 @@ public File downloadSingleFileUsingKeepWeb(String filePathName, String collectio
         return downloadedFile;
     }
 
+    public File downloadFileWithResume(String collectionUuid, String fileName, String pathToDownloadFolder, long offset, int bufferSize) throws IOException {
```

Comment:
It would be good to have a test for this new method.

Reply:
Done